napari/tests/test_numpy_like.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
```

Comment:
Should this file not include a `pytest.importorskip`?

Reply:
These packages are listed in our test dependencies so I don't think we need the `importorskip`, or am I mistaken?